aioasuswrt/asuswrt.py

```diff
@@ -255,18 +258,31 @@ def __init__(
         else:
             self.connection = SshConnection(host, port, username, password, ssh_key)
 
-    async def async_get_nvram(self, to_get):
+    async def async_get_nvram(self, to_get, use_cache=True):
         """Gets nvram"""
         data = {}
-        if to_get in GET_LIST:
+        if not (to_get in GET_LIST):
+            return data
+
+        now = datetime.utcnow()
```

Comment:
Probably better to use a monotonic time such as `time.monotonic` such that we don't get suprises due to system clock changes or winter/summer time

Reply:
Not a problem for me, but I used the same logic already implemented for other function (e.g. `async_get_connected_devices`) If we change here we should change also on other parts.

Reply:
Lets rework to time.monotonic in a separate PR, great pre-catch for that bug @JJdeVries